cmd/docker/docker.go

```diff
@@ -15,16 +15,18 @@ import (
 	pluginmanager "github.com/docker/cli/cli-plugins/manager"
 	"github.com/docker/cli/cli-plugins/socket"
 	"github.com/docker/cli/cli/command"
-	"github.com/docker/cli/cli/command/commands"
 	"github.com/docker/cli/cli/debug"
 	cliflags "github.com/docker/cli/cli/flags"
 	"github.com/docker/cli/cli/version"
 	platformsignals "github.com/docker/cli/cmd/docker/internal/signals"
+	"github.com/docker/cli/internal/commands"
 	"github.com/moby/moby/api/types/versions"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"go.opentelemetry.io/otel"
+
+	_ "github.com/docker/cli/cli/command/commands"
```

Comment:
Possibly this could even be moved to / merged with the internal/commands package, then `commands.AddCommands` could loop over the commands that were registered.